.github/workflows/release-check.yml

```
@@ -0,0 +1,43 @@
+name: Release Check
+
+on:
+  push:
+    branches:
+      - 'release/**'
+      - 'hotfix/**'
+
+env:
+  JAVA_VERSION: 19
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release-check-precondition:
+    name: Validate pushed commit to release/hotfix branch or pushed tag
```

Only the name could be changed:

```suggestion
    name: Validate pushed commit to release/hotfix branch
```

---

Fixed in https://github.com/cryptomator/cryptomator/pull/2535/commits/c7b181d08f19bb159a60b66fd03f9f662db04fcf